env/cpu/py3-master.yml

```diff
@@ -4,26 +4,39 @@ dependencies:
   - python=3.5
   - pip
   - perl
+  - pylint
```

Comment:
We can't use conda to install these packages for python 3.5 due to bugs in conda. Let's use pip instead.

But for simplicity, let's merge the Python 3.5 CI fixes as part of https://github.com/dmlc/gluon-nlp/pull/1023 Then you don't need to bother about them in this PR

Reply:
It looks like the master branch CI is failing with lots of numba errors

Reply:
That's due to the bug's in conda. Conda is not installing the most recent version of numba for some reason if you specify Python 3.5.

Just remove all changes to the CI environment files from your PR and it should pass.